### DISCUSSION TITLE
ثبت کارهای راهبران خبرنامه برای عدم تداخل

### DISCUSSION THREAD

**Discussion by ohmydevops** [softwaretalks/newsletter#281] [General]
<div dir='rtl'>

(پیش‌نویس)

از حالا به بعد زیر این پست، هر راهبر خبرنامه که بخواد کارهای خبرنامه رو شامل بررسی پست‌ها، لیبل‌زدن‌ها، و در نهایت ارسال‌شون رو انجام بده اینجا تا ساعت ۱۲ ظهر پنج‌شنبه بنویسه. اینطوری بقیه متوجه میشن و دیگه نیازی نیست اقدامی کنند. و اگر کسی تا ساعت ۱۲ وارد این صفحه شد و دید چیز خاصی برای اون روز اضافه نشده، میتونه مسئولیت اون روز رو به عهده بگیره و یادداشت بزاره برای بقیه

</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>
امروز پنجشنبه، ۳۱ تیر ۱۴۰۰، ارسال و تست خبرنامه رو به عهده میگیرم.
</div>

@Hameds

**Reply by ohmydevops** [reply]
<div dir='rtl'>
امروز پنجشنبه، ۰۷ مرداد ۱۴۰۰، ارسال و تست خبرنامه رو به عهده میگیرم.
</div>

@Hameds

**Reply by ohmydevops** [reply]
<div dir='rtl'>
امروز پنج شنبه، ۱۴ مرداد ۱۴۰۰ ارسال و تست خبرنامه رو به عهده میگیرم.
</div>

@Hameds

**Reply by ohmydevops** [reply]
<div dir='rtl'>

امروز پنج شنبه، ۲۱ مرداد ۱۴۰۰ ارسال و تست خبرنامه رو به عهده میگیرم.

</div>

@Hameds

**Reply by ohmydevops** [reply]
گویا مشکلی در زیرساخت پاکت وجود دارد. عصر مجددا تست میکنم

**Reply by ohmydevops** [reply]
امروز پنج شنبه - ۴ شهریور ۱۴۰۰ ارسال و تست خبرنامه رو به عهده میگیرم.

**Reply by Hameds** [reply]
من تازه اینو دیدم و به شکل عجیبی نوتیفکیشن این بخش برای من نیومده. آیا این کار همچنان ادامه داره؟

**Reply by ohmydevops** [reply]
سلام
من چون دیدم فعلا فقط خودم دارم ارسال میکنم گفتم اینجا چیزی نزارم. این برای زمانی خوب میشه که چند تا ادمین داریم

**Comment by ohmydevops** [comment]
سلام 
هفته بعد ارسال خبرنامه با من @Hameds 
(یعنی ۹ تیر ۱۴۰۱)

**Comment by ohmydevops** [comment]
اگر هم موافق هستی این دیسکاشن رو مجدد استفاده کنیم و معیار بزاریم. اینطوری از بقیه بچه ها هم میتونیم کمک بگیریم و یک سمافور داشته باشیم  برای عدم تداخل 

@Hameds

**Reply by Hameds** [reply]
امیرحسین جان شما از نظر ارسال خیالت راحت باشه. من هر پنجشنبه خبرنامه رو می‌فرستم. فعلاً تنها کار جمع آوری لینک‌هاست و تگ زدن‌شون.

**Reply by ohmydevops** [reply]
برای فردا شما ارسال میکنی؟

**Comment by ohmydevops** [comment]
سلام حامد جان خسته نباشی
من میتونم برای ماه پیش رو ارسال و تست خبرنامه رو به عهده بگیرم. اگر شما میخوای یک مدت پنج شنبه ها برای این موضوع تایم نزاری من میتونم پیش ببرمش مجدد

**Reply by Hameds** [reply]
سلام. ممنونم. من بخش ارسال رو می‌تونم انجام بدم. هر هفته توی تقویم تنظیم کردم و می‌فرستم. اگر فرصت بود در بخش محتوا با هم وقت بیشتری بگذاریم.

**Reply by ohmydevops** [reply]
سلام. حتما. چرا که نه.
